mev_inspect/abi.py

```diff
@@ -11,6 +11,21 @@
 THIS_FILE_DIRECTORY = Path(__file__).parents[0]
 ABI_DIRECTORY_PATH = THIS_FILE_DIRECTORY / "abis"
 
+# raw abi, for instantiating contract for queries (as opposed to classification, see below)
+def get_raw_abi(abi_name: str, protocol: Optional[Protocol]) -> Optional[str]:
+    abi_filename = f"{abi_name}.json"
+    abi_path = (
+        ABI_DIRECTORY_PATH / abi_filename
+        if protocol is None
+        else ABI_DIRECTORY_PATH / protocol.value / abi_filename
+    )
+
+    if abi_path.is_file():
+        with abi_path.open() as abi_file:
+            return abi_file.read()
+
+    return None
```

Comment:
I think this got lost in the changes, but does `get_abi(...).json()` work instead?

If not, it'd be better for the two to share underlying logic rather than duplicating.

ex:
```
def get_abi(abi_name: str, protocol: Optional[Protocol]) -> Optional[ABI]:
    abi_json = get_raw_abi(abi_name, protocol)
    if abi_json is None:
        return None
    else:
        return parse_obj_as(ABI, abi_json)
```

Reply:
@lukevs yeah, the .json() didn't work but I can remove the duplication with shared functions